File: neetcode/problems/leetcode1929/solution.py
from typing import List


class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        new_arr = [0] * len(nums) * 2
        for i in range(len(nums)):
            new_arr[i] = nums[i]
            new_arr[len(nums) + i] = nums[i]

        return new_arr

File: neetcode/problems/leetcode1929/solution.js
const getConcatenation = nums => {
	const newArr = Array.from({ length: nums.length * 2 }, _ => 0);

	for (let i = 0; i < nums.length; i++) {
		newArr[i] = nums[i];
		newArr[nums.length + i] = nums[i];
	}

	return newArr;
};
